### ISSUE TITLE
Fix all fortitude C131 issues

### ISSUE THREAD

**Issue by awvwgk** [dftd3/simple-dftd3#204]
Default module accessibility

<details>

````
src/dftd3/cutoff.f90:17:1: C131 module 'dftd3_cutoff' missing default accessibility statement
   |
15 | ! along with s-dftd3.  If not, see <https://www.gnu.org/licenses/>.
16 |
17 | module dftd3_cutoff
   | ^^^^^^^^^^^^^^^^^^^ C131
18 |    use mctc_env, only : wp
19 |    implicit none
   |

src/dftd3/damping.f90:17:1: C131 module 'dftd3_damping' missing default accessibility statement
   |
15 | ! along with s-dftd3.  If not, see <https://www.gnu.org/licenses/>.
16 |
17 | module dftd3_damping
   | ^^^^^^^^^^^^^^^^^^^^ C131
18 |    use mctc_env, only : wp
19 |    use mctc_io, only : structure_type
20 |    implicit none
   |

src/dftd3/damping/atm.f90:17:1: C131 module 'dftd3_damping_atm' missing default accessibility statement
   |
15 | ! along with s-dftd3.  If not, see <https://www.gnu.org/licenses/>.
16 |
17 | module dftd3_damping_atm
   | ^^^^^^^^^^^^^^^^^^^^^^^^ C131
18 |    use dftd3_cutoff, only : smooth_cutoff
19 |    use mctc_env, only : wp
20 |    use mctc_io, only : structure_type
   |

src/dftd3/damping/cso.f90:17:1: C131 module 'dftd3_damping_cso' missing default accessibility statement
   |
15 | ! along with s-dftd3.  If not, see <https://www.gnu.org/licenses/>.
16 |
17 | module dftd3_damping_cso
   | ^^^^^^^^^^^^^^^^^^^^^^^^ C131
18 |    use dftd3_cutoff, only : smooth_cutoff
19 |    use dftd3_damping, only : damping_param
20 |    use dftd3_damping_atm, only : get_atm_dispersion, get_atm_pairwise_dispersion
   |

src/dftd3/damping/mzero.f90:17:1: C131 module 'dftd3_damping_mzero' missing default accessibility statement
   |
15 | ! along with s-dftd3.  If not, see <https://www.gnu.org/licenses/>.
16 |
17 | module dftd3_damping_mzero
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^ C131
18 |    use dftd3_cutoff, only : smooth_cutoff
19 |    use dftd3_damping, only : damping_param
20 |    use dftd3_damping_atm, only : get_atm_dispersion, get_atm_pairwise_dispersion
   |

src/dftd3/damping/optimizedpower.f90:17:1: C131 module 'dftd3_damping_optimizedpower' missing default accessibility statement
   |
15 | ! along with s-dftd3.  If not, see <https://www.gnu.org/licenses/>.
16 |
17 | module dftd3_damping_optimizedpower
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ C131
18 |    use dftd3_cutoff, only : smooth_cutoff
19 |    use dftd3_damping, only : damping_param
20 |    use dftd3_damping_atm, only : get_atm_dispersion, get_atm_pairwise_dispersion
   |

src/dftd3/damping/rational.f90:17:1: C131 module 'dftd3_damping_rational' missing default accessibility statement
   |
15 | ! along with s-dftd3.  If not, see <https://www.gnu.org/licenses/>.
16 |
17 | module dftd3_damping_rational
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ C131
18 |    use dftd3_cutoff, only : smooth_cutoff
19 |    use dftd3_damping, only : damping_param
20 |    use dftd3_damping_atm, only : get_atm_dispersion, get_atm_pairwise_dispersion
   |

src/dftd3/damping/z.f90:17:1: C131 module 'dftd3_damping_z' missing default accessibility statement
   |
15 | ! along with s-dftd3.  If not, see <https://www.gnu.org/licenses/>.
16 |
17 | module dftd3_damping_z
   | ^^^^^^^^^^^^^^^^^^^^^^ C131
18 |    use dftd3_cutoff, only : smooth_cutoff
19 |    use dftd3_damping, only : damping_param
20 |    use dftd3_damping_atm, only : get_atm_dispersion, get_atm_pairwise_dispersion
   |

src/dftd3/damping/zero.f90:17:1: C131 module 'dftd3_damping_zero' missing default accessibility statement
   |
15 | ! along with s-dftd3.  If not, see <https://www.gnu.org/licenses/>.
16 |
17 | module dftd3_damping_zero
   | ^^^^^^^^^^^^^^^^^^^^^^^^^ C131
18 |    use dftd3_cutoff, only : smooth_cutoff
19 |    use dftd3_damping, only : damping_param
20 |    use dftd3_damping_atm, only : get_atm_dispersion, get_atm_pairwise_dispersion
   |

src/dftd3/utils.f90:17:1: C131 module 'dftd3_utils' missing default accessibility statement
   |
15 | ! along with s-dftd3.  If not, see <https://www.gnu.org/licenses/>.
16 |
17 | module dftd3_utils
   | ^^^^^^^^^^^^^^^^^^ C131
18 |    use mctc_env, only : wp
19 |    use mctc_io_math, only : matinv_3x3
20 |    implicit none
````
